AdaptivePELE/freeEnergies/estimateDG.py

```diff
@@ -120,11 +120,11 @@ def copyWorkingTrajectories(fileWildcard, length=None, ntrajs=None, bootstrap=Tr
         writenFiles.append(dst)
         traj = np.loadtxt(trajFile)
         if length is None:
-            traj_len = len(traj)  # so that later eveything is copied
-        else:
-            traj_len = length
+            traj_len = length =len(traj)  # so that later eveything is copied
```

Comment:
This change reintroduces the boostrap bug related to trajectories length. Since the length of the trajectories is only considered if length is None, when length is set to a number (the firs iteration of the for loop) then length will never change and all trajectories will be cut at the length of the first, that is why I introduced the local variable traj_len